### ISSUE TITLE
fermata mod app for android 14

### ISSUE THREAD

**Issue by ishanthap** [AndreyPavlenko/Fermata#450]
hi, is it true that someone is modifying the fermata apk to work well with Android 14. found several videos in YouTube with this. he charges $5, is it fake?

**Comment by AndreyPavlenko** [comment]
I'm not aware of that.

**Comment by luitruda** [comment]
Hi, unfortunately I also updated my Xiaomi 12 to hiperos and nothing doesn't work anymore fermata auto can someone help me otherwise I have to do a lot of work to downgrade to miu14 thanks for your answers

**Comment by IOCMX33** [comment]
Es correcto adjunto link del canal de YouTube 
Donde se indica lo mencionado 
https://youtu.be/rOtsMZsD3RY?si=91bk0C-EMfA99pZa
![Uploading Screenshot_20240324_175051_YouTube.jpg…]()

**Comment by ishanthap** [comment]
yes this was the one. does this work on non rooted phones, anyone tried his method or mod apk?

**Comment by akulp** [comment]
It looks like spam for me... :)

**Comment by luitruda** [comment]
yes I agree they promise things they do only to extort money

**Comment by jushi-j** [comment]
using the paid app, head unit reloaded, It can initialize a self-mode for Android Auto which has the fermada mirror option. once you choose the formata mirror or option you can plug in the USB to the car and this works for my Google Pixel 6 with Android 14 and my Nissan Murano 2020.

**Comment by Alejandro2454** [comment]
@jushi-j ¿cuál es el nombre de la paid app?

**Comment by ishanthap** [comment]
Best option is to use an old android phone, in my case samsung S6 on lineageOS 18.1 root by magisk and installed AA with Fermata works well on youtube and other functions.
S6 connected to Peugeot 3008 via usb cable and bluetooth disabled by service freezer apk so that only main phone connected for calls and messages and S6 for streaming videos/fermata/audio and google maps.
